Providers/Profile.swift

```diff
@@ -499,7 +503,7 @@ open class BrowserProfile: Profile {
     }
 
     lazy var logins: BrowserLogins & SyncableLogins & ResettableSyncStorage = {
-        return SQLiteLogins(db: self.loginsDB)
+        return SQLiteLogins(db: self.loginsDB!)
```

Comment:
It's only not safe when in extensions, which really don't want to be accessing `logins`.

I thought about making `logins` optional, but it's a reasonably big change. We could also use a `MockLogins`??

```swift
        guard let loginsDB = self.loginsDB else {
            return MockLogins(files: FileAccessor(rootPath: "disposable-extension"))
        }
        return SQLiteLogins(db: loginsDB)
```
(This feels ridiculous!)